adodb.inc.php

```diff
@@ -837,7 +842,7 @@ protected function parseHostNameAndPort() {
 	 *
 	 * @return bool
 	 */
-	function Connect($argHostname = "", $argUsername = "", $argPassword = "", $argDatabaseName = "", $forceNew = false) {
+	function _connect($argHostname = "", $argUsername = "", $argPassword = "", $argDatabaseName = "", $forceNew = false) {
```

Comment:
You can't do that ! 

connect() is ADOdb's main connection public method, which relies on driver-specific, protected _connect().

All changes to connect(), nconnect(), pconnect() must be undone.